package.json

```diff
@@ -136,8 +138,7 @@
         "test:command-reference",
         "test:deprecation-policy",
         "lint",
-        "test:json-schema",
-        "link-check"
```

Comment:
link-check seems to want a published CLI command which we don't have yet obviously

Reply:
Removing for now